recipes/mmcv/conda_build_config.yml

```diff
@@ -0,0 +1,2 @@
+pytorch:
+  - 1.11
```

Comment:
Do you actually need this file?

Reply:
this file will stop you from migrating to newer versions.

Reply:
@hmaarrfk , as @ngam mentioned MMCV ([upstream](https://mmcv.readthedocs.io/en/latest/get_started/installation.html)) is built for all  combinations of python/cuda/torch. Assuming we should do the same having just `1.11` for now.
My reasoning was:
1. If we don't pin it during build - it's quite opaque what version it was built against.
2. Whenever new pytorch is released - any further re-build of this stock will pick it up without a guarantee of compatibility.

What do you suggest?

Reply:
once you rerender, ci support files are generated losying the specific versions of the dependencies.

in my experience, getting the bots help is critical when you maintain mkre than about 5 packages.